backend/src/nodes/onnx_nodes.py

```diff
@@ -173,3 +175,22 @@ def run(
             in_nc,
             lambda i: self.upscale(i, session, split_factor, change_shape),
         )
+
+
+@NodeFactory.register("chainner:onnx:convert_to_ncnn")
+class ConvertOnnxToNcnnNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = """Convert an ONNX model to NCNN."""
+        self.inputs = [OnnxModelInput("ONNX Model"), OnnxFpDropdown()]
+        self.outputs = [NcnnModelOutput("NCNN Model")]
+
+        self.category = ONNXCategory
+        self.name = "Convert To NCNN"
+        self.icon = "NCNN"
+        self.sub = "Utility"
+
+    def run(self, onnx_model: bytes, is_fp16: int) -> NcnnModel:
+        converter = Onnx2NcnnConverter(onnx.load_model_from_string(onnx_model))
+        ncnn_model = converter.convert(is_fp16, False)
```

Comment:
Please convert `is_fp16` to a bool before passing it in. The correctness of `Onnx2NcnnConverter.convert` depends on the values of `OnnxFpDropdown` right now. That's not good.

Reply:
Isn't it going to do that regardless? If I convert it to a bool, it will still depend on the values of OnnxFpDropdown.

Reply:
> Isn't it going to do that regardless?

No. The contract `convert` defines now (after your recent commits) is decoupled from the `OnnxFpDropdown`. If we change the values of `OnnxFpDropdown`, we won't have to change `convert`'s implementation, but the `bool()` cast. So future changes are going to be a lot more localized.

I didn't like that `convert` made assumptions about the values of `OnnxFpDropdown` (global reasoning). These two things should be independent of each other.